src/dotnet/UI.Blazor/Components/Modal/BlazoredModal/BlazoredModal.razor

```diff
@@ -0,0 +1,165 @@
+﻿@using System.Collections.ObjectModel
+@using Microsoft.AspNetCore.Components.Routing
+@namespace Blazored.Modal
+
+@inject NavigationManager NavigationManager
+@inject IJSRuntime JsRuntime
```

Comment:
@crui3er why, why to use 3rd different name for the same thing?
- `JS` is what we prefer
- You were already using `JSRuntime`
- and now you add `JsRuntime`

What's the point of coding style standards? To reduce the variance - if you see X, you must be certain it's X. Now, if you add Y and Z options, you kinda defeat the whole reason to continue enforcing this, coz it's a signal for everyone else there is no rule & any other name is fine.

That's why I consistently use `services` & few other short names - if something is frequently used, readability (i.e. short unique name) is way more important when in a case with some random / rare name.  This is why I don't care that much about other names. But frequent names are important, and it's important we use the same var name for these.